en/sim_gazebo_gz/README.md

```diff
@@ -218,16 +283,35 @@ To add a new model:
      PX4_SYS_AUTOSTART=<your new airframe id> ./build/px4_sitl_default/bin/px4
      ```
 
+1. Add CMake Target for the [airframe](https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/init.d-posix/airframes/CMakeLists.txt).
+
+   - If you plan to use "regular" mode, add your model SDF to `Tools/simulation/gz/models/`.
+   - If you plan to use _standalone_ mode, add your model SDF to `~/.simulation-gazebo/models/`
```

Comment:
The rest of this is about how you add models to PX4. The assumption being that after this, someone else who wants to follow the instructions will be able to use the models themselves after downloading a version with this airframe and CMAKEList change after the models repo has been updated.

However, then we have these bullet points that say you have to copy the files to particular locations. You only need to do this when you're testing. 

So I am not going to wait on this, but do you think it makes sense to say something like. Once the models are updated to the repo they will be automatically to the right location when you install PX4. When testing you can copy them manually .... then the bullets ?